### DISCUSSION TITLE
ip address is not associated with the VM

### DISCUSSION THREAD

**Discussion by north13dev** [bb-Ricardo/netbox-sync#354] [Q&A]
Hello! I have a problem, ip address is not associated with the VM, the address is created in a strange way, if you click on it it says there is no such thing, but there is no binding of the address to the VM, it’s just marked as busy
![scrnli_01-11-2023_14-49-52](https://github.com/bb-Ricardo/netbox-sync/assets/46131067/ff42ffff-5809-4b23-837a-0111582d4e38)

**Comment by bb-Ricardo** [comment]
Hi, this looks very odd. Did you try running the latest NetBox version?

What did the log of NetBox sync say about the creation of this VM?

**Reply by north13dev** [reply]
Thanks, I updated netbox to version 3.6.4 and it worked!
This is a bug in version 3.6.0

**Reply by bb-Ricardo** [reply]
Ahh, great you got it working. Good to know.
